### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by PuneetGurjar** [efroemling/ballistica#215]
Why don't pc players have arrow icon in front of their player whereas in case of android devices it is present
That arrow helps in better aiming.
![ANDOIRD](https://user-images.githubusercontent.com/68325952/102180665-40212b00-3ecf-11eb-8a76-82e7d4a4de9d.PNG)

**Comment by efroemling** [comment]
The arrow being present for the touchscreen-controlled player is mostly to compensate for the fact that its harder to tell where your joystick currently is on a touchscreen, while with analog physical controls you can mostly 'feel' it. Perhaps someone could add an advanced option to show arrows for all players though (or all local players at least). I'd be happy to point out what would need to be changed if someone wants to try.

**Comment by Freaku17** [comment]
I agree, playing in some modded gamemodes. The tint is reduced to 0, and then it's impossible to ever locate where my cute spaz is .... 😞
